overlord/ifacestate/helpers.go

```diff
@@ -888,6 +888,22 @@ func setHotplugChangeAttrs(chg *state.Change, seq int, hotplugKey string) {
 	chg.Set("hotplug-key", hotplugKey)
 }
 
+// addHotplugSeqWaitTask sets mandatory hotplug attributes on the hotplug change, adds "hotplug-seq-wait" task
+// and makes all existing tasks of the change wait for it.
```

Comment:
the comment should tell a bit more what's the effect of this I think for the change

Reply:
I merged the PR, please do this in a follow up @stolowski